lib/components/canvas/index.ts

```diff
@@ -106,19 +106,20 @@ export class CanvasSink extends Sink {
     }
     let updateInfo: (info: RateInfo, update: ByteDuration) => void
 
-    let ctx = el.getContext('bitmaprenderer')
+    // The createImageBitmap function is supported in Chrome and Firefox
+    // (https://developer.mozilla.org/en-US/docs/Web/API/WindowOrWorkerGlobalScope/createImageBitmap)
+    // Note: drawImage can also be used instead of transferFromImageBitmap, but it caused
+    // very large memory use in Chrome (goes up to ~2-3GB, then drops again).
+    // Do do not call el.getContext twice, safari returns null for second call
+    let ctx = window.createImageBitmap ? el.getContext('bitmaprenderer') : null;
     let drawImageBlob: BlobMessageHandler
     if (ctx !== null) {
-      // The createImageBitmap function is supported in Chrome and Firefox
-      // (https://developer.mozilla.org/en-US/docs/Web/API/WindowOrWorkerGlobalScope/createImageBitmap)
-      // Note: drawImage can also be used instead of transferFromImageBitmap, but it caused
-      // very large memory use in Chrome (goes up to ~2-3GB, then drops again).
       drawImageBlob = ({ blob }) => {
         info.renderedFrames++
         window
           .createImageBitmap(blob)
           .then(imageBitmap => {
-            ;(ctx as any).transferFromImageBitmap(imageBitmap)
+            ; (ctx as any).transferFromImageBitmap(imageBitmap)
```

Comment:
Did you run `prettier` on this file? We use `prettier` to handle any formatting so it stays consistent.

Reply:
Well, I have plugins on editor that should follow tslint, eslint, editor and prettier rules.. Editor automatically updated this.. I'll run it later manually and update PR.